doc/classes/EditorSettings.xml

```diff
@@ -448,6 +448,12 @@
 		<member name="editors/visual_editors/visual_shader/port_preview_size" type="int" setter="" getter="">
 			The size to use for port previews in the visual shader uniforms (toggled by clicking the "eye" icon next to an output). The value is defined in pixels at 100% zoom, and will scale with zoom automatically.
 		</member>
+		<member name="export/android/use_wifi_for_remote_debug" type="bool" setter="" getter="">
+			If [code]true[/code], then remote debugging will be performed over Wi-Fi. If [code]false[/code], remote debugging over USB will be used when it is supported by the android API level (version 21 or higher)
```

Comment:
These won't be in the documentation unless they are defined outside of the android only part, as it is run on linux when generating

Reply:
So I should just remove them? Is there somewhere else I should add them to the documentation?

Reply:
Don't know but the other specific settings there aren't documented either, so would be best to do in a separate PR if required

Reply:
@Pingar5 You can build the engine, and run `godot --doctool` with the generated build to update the documentation entries.